src/presetsController.ts

```diff
@@ -41,7 +41,11 @@ export class PresetsController {
     static async init(cmakeProject: CMakeProject, kitsController: KitsController): Promise<PresetsController> {
         const presetsController = new PresetsController(cmakeProject, kitsController);
         const expandSourceDir = async (dir: string) => {
-            const workspaceFolder = cmakeProject.folder.uri.fsPath;
+            let workspaceFolder = cmakeProject.folder.uri.fsPath;
+            if (workspaceFolder.length > 1 && workspaceFolder.charCodeAt(0) > 97 && workspaceFolder.charCodeAt(0) <= 122 && workspaceFolder[1] === ':') {
+                // Windows drive letter should be uppercase, for consistency with other tools like Visual Studio.
+                workspaceFolder = workspaceFolder[0] + workspaceFolder[1].toUpperCase() + workspaceFolder.slice(2);
```

Comment:
I'm also thinking that it would be best to modify the normalization functions in the `util` module instead of having a one-off here.  I'll propose an alternate change.

Reply:
Yes you are right, my mistake here